src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4910,36 +4910,40 @@ components:
           example: false
           default: false
     Endpoint:
+      allOf:
```

Comment:
The `allOf` keyword, combined with the OpenAPI model generator that we are using, seems to generate a redundant `EndpointAllOf` object. We have mainly tried to avoid these and created for example a `ClientAdd` and `ServiceDescriptionUpdate` models that only have the allowed and required fields in them. So maybe consider separate models (without `allOf`) for these endpoint schemas.

There is a task ([XRDDEV-668](https://jira.niis.org/browse/XRDDEV-668)) about using `readOnly` keyword in those fields that aren't supposed to be sent in a request. So maybe some unifying methods should be considered and documented when that task is being investigated further. **Spoiler alert:** The `readOnly` flag has no effect on anything

Reply:
Removed allOf -operator and renamed EndpointPathAndMethod to EndpointUpdate.